src/chrome/content/rules/Bundesdruckerei.de.xml

```diff
@@ -0,0 +1,37 @@
+<!--
+	Connection reset:
+		helpdesk.bundesdruckerei.de
+
+	Empty response:
+		mis.bundesdruckerei.de
+
+	Requires client certificate:
+		ems.bundesdruckerei.de
+		service.bundesdruckerei.de
```

Comment:
Actually, you can redirect these too, since they don't work over HTTP.

Reply:
They won't pass fetch tests, so I'm okay with having them in comments.